config/locales/en.yml

```
@@ -100,7 +100,7 @@ en:
         unauthorized_client: 'The client is not authorized to perform this request using this method.'
         access_denied: 'The resource owner or authorization server denied the request.'
         invalid_scope: 'The requested scope is invalid, unknown, or malformed.'
-        invalid_code_challenge_method: 'The code challenge method must be plain or S256.'
+        invalid_code_challenge_method: 'The code challenge method must be one of %{challenge_methods}.'
```

This could probably use some tweaking, just let me know how you think this should read.

---

You'd probably want to use the one / other syntax here.. 
```suggestion
        invalid_code_challenge_method:
          one: 'The code challenge method must be %{challenge_methods}.'
          other: 'The code challenge method must be one of %{challenge_methods}.'
```

Which would need a `count` on `challenge_methods`, iirc.

---

Awesome suggestions, I went ahead and updated in ccc97732f3977e57d3a7c541689c30f418de6697